### ISSUE TITLE
We need to replicate the realtime reports in  desktop

### ISSUE THREAD

**Issue by timlinux** [inasafe/inasafe#4724]

**Comment by timlinux** [comment]
* Ash fall
* Flood in Jakarta
* EQ

**Comment by ivanbusthomi** [comment]
This is the first test realtime report of volcanic ash using InaSAFE v.4.3
![ash_v1](https://user-images.githubusercontent.com/7351977/35207284-aaf786e2-ff74-11e7-89b6-97cf9c091fdd.png)

there are some elements that still need to be updated, but the information shown there are already based on the analysis.

**Comment by ivanbusthomi** [comment]
This is the first test realtime report of earthquake using InaSAFE v.4.3

![earthquake_replicate](https://user-images.githubusercontent.com/7351977/35272139-d5ae9a72-0066-11e8-8a0e-2f5fbee1a909.png)

**Comment by ivanbusthomi** [comment]
This is the first test realtime report of flood using InaSAFE v.4.3
there is some information at the top of the report that still missing, maybe we can add that information from the website side?
cc @ismailsunni @myarjunar @lucernae 

![flood_v1](https://user-images.githubusercontent.com/7351977/35318727-3a46a1de-010f-11e8-8125-5ac1c24c7843.png)

**Comment by ivanbusthomi** [comment]
Hi @Gustry and @myarjunar - this is what I found a little bit strange.
so basically I used two different expression, utilizing information from two different analysis summary layer:
* fatalities information from population analysis summary layer - using `inasafe_impact_analysis_field_value('fatalities') `, 
* nearby places information from place analysis summary layer using `name_of_the_nearest_place()`

when I tried selecting either layer in the layers panel, only one of the information is shown at a time.

![composer](https://user-images.githubusercontent.com/7351977/35322074-89c89fd0-011b-11e8-89fd-31a756d10f97.gif)

**Comment by ivanbusthomi** [comment]
This is the project files:
📦 [Realtime ash](https://github.com/inasafe/inasafe/files/1659969/realtime_ash.zip)
📦 [Realtime earthquake](https://github.com/inasafe/inasafe/files/1659970/realtime_earthquake.zip)
📦 [Realtime flood](https://github.com/inasafe/inasafe/files/1659971/realtime_flood.zip)

Each of them contains the qpt templates. Each project file also contains a template in the composer manager.

**Comment by ismailsunni** [comment]
Hi @ivanbusthomi 
It seems you also need date, time, and flood ID for the flood report, right? I think we should add that in the extra keyword also.

**Comment by ivanbusthomi** [comment]
Hi @ismailsunni - yes, I forgot that we also have that element in flood. is it easy to add? My initial thought was using current workflow in realtime at the moment where it is swapped with the proper value in website side. But It'd be nice if we can implement similar workflow using extra keywords like the other two.
I am sorry for missing that bit before the release.

**Comment by lucernae** [comment]
It needs to be added in the desktop side as an extra keywords to actually store the value. The website can only pass the value to hazard file, but can't include it in the analysis by it self to generate report.
An additional reason is, we also need the extra keywords for Desktop user to pass the value. Because they won't get them from the website.

**Comment by ismailsunni** [comment]
@lucernae 
yes, I already added the flood one. And the other extra keywords in https://github.com/inasafe/inasafe/issues/4857

**Comment by ivanbusthomi** [comment]
Thank you @ismailsunni for the addition of the hazard extra keywords for the flood hazard. And I agree with @lucernae that these extra keywords will also be useful for the user that download the data and run the analysis on the desktop.

**Comment by ivanbusthomi** [comment]
**WIP** documents of the documentation in creating the template
[Template documentation](https://docs.google.com/document/d/121BYj0o78ka9VXs9pzuV9HsZ-Mc4VdcS1wnOERArhLQ/edit?usp=sharing)

**Comment by timlinux** [comment]
@ivanbusthomi Wow just cheering you on from the sides here - the reports you made look really great!

**Comment by ismailsunni** [comment]
btw @timlinux 
Perhaps we should also make InaSAFE realtime to upload not only the .qpt but also another files to re-create a custom report.

**Comment by Gustry** [comment]
like organisation, north arrow etc I guess

**Comment by ivanbusthomi** [comment]
This template is still receiving some feedback and not final yet.

**Comment by ivanbusthomi** [comment]
Hi @timlinux, @Charlotte-Morgan, @myarjunar @ismailsunni @lucernae 
This is the latest state of the realtime template:
* Earthquake
![image](https://user-images.githubusercontent.com/7351977/38067706-f1b464c8-3337-11e8-8389-734d1e43ed92.png)

* Flood
![image](https://user-images.githubusercontent.com/7351977/38067729-04aab9c4-3338-11e8-8858-b6ef83d49c74.png)

* Ash
![image](https://user-images.githubusercontent.com/7351977/38068262-6e0d4722-333a-11e8-8639-0316eed7f298.png)
